devtools/conda-envs/examples_env.yml

```diff
@@ -12,11 +12,14 @@ dependencies:
     - packaging
     # Examples
     - openff-toolkit-examples
-    - openff-interchange==0.3.5 # Pin Interchange 0.3.5 because 0.3.6 has a broken example
+    - openff-interchange # 0.3.6 has a broken example, but we're loading examples from the central-examples branch where it's fixed
```

Comment:
0.3.7 is released, which should work, and with any luck the tarballs will actually be online in an hour or two from me writing this comment